src/win32/wazuh-installer.wxs

```diff
@@ -177,10 +177,17 @@
                         <File Id="LOCAL_INTERNAL_OPTIONS.CONF" Name="local_internal_options.conf" Source="default-local_internal_options.conf" KeyPath="yes" />
                     </Component>
 
+                    <Component Id="OSSEC_PRE6.CONF" DiskId="1" Guid="71947F0E-56D6-4E14-8539-214dDFF6CA2BE" NeverOverwrite="yes" Permanent="yes">

```

Comment:
It won't work, the GUID has an extra character. _71947F0E-56D6-4E14-8539-214**d**DFF6CA2BE_ -> _71947F0E-56D6-4E14-8539-214DFF6CA2BE_